packages/worker/src/app.service.spec.ts

```diff
@@ -215,10 +215,10 @@ describe("AppService", () => {
 
       expect(blocksRevertService.handleRevert).toBeCalledWith(blockNumber);
 
-      expect(blockService.start).toBeCalledTimes(1);
-      expect(batchService.start).toBeCalledTimes(1);
-      expect(counterService.start).toBeCalledTimes(1);
-      expect(balancesCleanerService.start).toBeCalledTimes(1);
+      expect(blockService.start).toBeCalledTimes(2);
+      expect(batchService.start).toBeCalledTimes(2);
+      expect(counterService.start).toBeCalledTimes(2);
+      expect(balancesCleanerService.start).toBeCalledTimes(2);
```

Comment:
Honestly, not sure how it worked before but these checks started to fail after`jest` update. 2 - is the correct number of calls here, since 1st call happens on app init, and then the 2nd - while processing the revert event.